tsl/src/chunk_api.c

```diff
@@ -1627,3 +1627,63 @@ chunk_api_get_chunk_relstats(PG_FUNCTION_ARGS)
 {
 	return chunk_api_get_chunk_stats(fcinfo, false);
 }
+
+TS_FUNCTION_INFO_V1(ts_chunk_get_dimension_slices);
+Datum
+ts_chunk_get_dimension_slices(PG_FUNCTION_ARGS)
```

Comment:
Is this function needed?

There's already an internal `show_chunk` function that shows the slices:

```
nyc_data=# select slices from _timescaledb_internal.show_chunk('_timescaledb_internal._dist_hyper_1_150_chunk');
                                                    slices                                                    
--------------------------------------------------------------------------------------------------------------
 {"payment_type": [1073741823, 9223372036854775807], "pickup_datetime": [1452124800000000, 1452729600000000]}
(1 row)

```

Reply:
Good point! I will look into it.

Reply:
Replaced with the suggestion.